client/my-sites/theme/main.jsx

```diff
@@ -1267,7 +1237,7 @@ class ThemeSheet extends Component {
 					onClick={ null === onClick ? noop : onClick }
 					href={ plansUrl }
 					showIcon={ true }
-					forceDisplay={ forceDisplay }
```

Comment:
This is introducing a regression for logged-in users. For instance, the upsell banner shouldn't be visible on WooCommerce Themes for sites on the Business plan.